_posts/troubleshooting/2018-11-21-cors.md

```diff
@@ -0,0 +1,28 @@
+---
+date: 2018-11-21
+title: Troubleshooting Gate/Deck CORS Issues
+categories:
+   - troubleshooting
+description: Troubleshooting Gate/Deck CORS Issues
+type: Document
+---
+
+If you're trying to hook up Spinnaker Gate and Deck and using different hostnames/domains for Gate and Deck, and you're experiencing CORS issues (where Gate is refusing refusing connections from Deck), you may need to modify Gate to support CORS from your Deck fqdn.
+
+For example, if you're using gate.domain.com for your Gate, and spinnaker.domain.com for your Deck, then go into your halconfig (`.hal/config`), into the relevant security section of your halyard deployment configuration, and add the `corsAccessPattern` field, like this:
+
```

Comment:
add `backticks` around `gate.domain.com` and `spinnaker.domain.com` so that it has some emphasis on it.

Reply:
Capitalize Halyard.